### PR TITLE
feat(x/ecocredit): add classes to basket query response

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Closes: #793

Add classes to basket query response.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/ecocredit/server/basket/query_basket.go [hunk 1]
```diff
@@ -24,5 +25,28 @@ func (k Keeper) Basket(ctx context.Context, request *baskettypes.QueryBasketRequ
...
+			return nil, err
+		}
+
+		basketClassGogo := &baskettypes.BasketClass{}
+		err = PulsarToGogoSlow(class, basketClassGogo)
+		if err != nil {
+			return nil, err
+		}
+
+		classes = append(classes, basketClassGogo.ClassId)
```
Comment (aaronc) [inline x/ecocredit/server/basket/query_basket.go:46]:
```suggestion
		classes = append(classes, class.ClassId)
```
don't need to convert to gogo

**Comment by ryanchristo** [comment]
Pending #802 and api release.

**Comment by ryanchristo** [comment]
Manually tested using the CLI.

```
$ ./build/regen q ecocredit basket eco.uC.NCT
basket:
  basket_denom: eco.uC.NCT
  credit_type_abbrev: C
  date_criteria: null
  disable_auto_retire: false
  exponent: 6
  id: "1"
  name: NCT
classes:
- C01
- C02
```

**Comment by technicallyty** [comment]
even though there was a manual test, i think it would be valuable to add a simple unit test here

**Review by technicallyty — APPROVED** [review]

**Review by clevinson — APPROVED** [review]

File: x/ecocredit/server/basket/query_basket_test.go [hunk 2]
```diff
@@ -34,3 +34,31 @@ func TestKeeper_Basket(t *testing.T) {
...
+	require.NoError(t, err)
+
+	// query
+	res, err := s.k.Basket(s.ctx, &baskettypes.QueryBasketRequest{
+		BasketDenom: basketDenom,
+	})
+	require.NoError(t, err)
+	require.Equal(t, basketDenom, res.Basket.BasketDenom)
+	require.Equal(t, []string{classId}, res.Classes)
+}
```
Comment (robert-zaremba) [inline x/ecocredit/server/basket/query_basket_test.go:64]: I think we can merge the 2 tests above - no need to duplicate most o the code.

Also, ideally would be great to make a test on the RPC server level (without running the server). We can do it in a separate PR.

**Review by robert-zaremba — APPROVED** [review]
